alerts/spider.md

```diff
@@ -0,0 +1,10 @@
+---
+title: "Itho Daalderop Spider"
+created: 2023-09-01 00:00:00
+integrations:
+  - spider
+homeassistant: ">2023.9.0"
+---
+
+The Itho Daalderop Spider web application will stop working as of September 1st 2023. As the integration depends on this web application, it will stop working.
+There is no alternative solution.
```

Comment:
This will create an active repair issue on the system of users. We need to tell the action they have to take to resolve the issue showing in their repair centre.

../Frenck